.github/workflows/newman.yml

```
@@ -21,7 +21,14 @@ on:
         required: true
       registry_password:
         required: true
-
+      CLIENT_ID :
+        required: true
+      CLIENT_SECRET:
+        required: true
+      JIRA_USER:
+          required: true
+      JIRA_TOKEN_TEST:
+          required: true

```

for consistency with other resuable workflow, use secret in lowercase
```suggestion
      client_id:
        required: true
      client_secret:
        required: true
      jira_user:
          required: true
      jira_token_test:
          required: true

```